build/walrus.cmake

```diff
@@ -68,6 +68,7 @@ ELSEIF (${WALRUS_MODE} STREQUAL "release")
 ENDIF()
 
 SET (WABT_DEFINITIONS ${WALRUS_DEFINITIONS})
+SET (WITH_EXCEPTIONS TRUE)
```

Comment:
Where is this flag used for?

Reply:
I want to throw exception in wabt code, but wabt don't enable exception feature default.

Reply:
I see :)